containerstats-pipeline.md

```diff
@@ -1,10 +1,10 @@
 # ContainerStats Pipeline
 
-The ContainerStats pipeline builds statistical representations of individual
+The ContainerStats pipeline builds histograms, statistical representations of individual
```

Comment:
For now, let's leave this detail out. It's entirely possible for us to add other representations in the future and histograms are mostly an implementation detail.